source/community/reactnative/src/models/model-factory.js

```diff
@@ -21,7 +21,11 @@ export class ModelFactory {
             }
         } else {
             const elementModel = this.getElement(payload, parent);
-            return this.addCustomPropertyInModel(elementModel, payload)
+            if (elementModel) {
```

Comment:
Please add element null check in requirements satisfied case too.

Reply:
Added element null check for requirements satisfied case also.